pycode/memilio-simulation/memilio/simulation/osecir.cpp

```diff
@@ -222,20 +225,20 @@ PYBIND11_MODULE(_simulation_secir, m)
         [](double t0, double tmax, double dt, const mio::osecir::Model<double>& model) {
             return mio::osecir::simulate(t0, tmax, dt, model);
         },
-        "Simulates a Secir Model1 from t0 to tmax.", py::arg("t0"), py::arg("tmax"), py::arg("dt"), py::arg("model"));
+        "Simulates an ODE SECIHURD model from t0 to tmax.", py::arg("t0"), py::arg("tmax"), py::arg("dt"), py::arg("model"));
 
     m.def(
         "simulate_flows",
         [](double t0, double tmax, double dt, const mio::osecir::Model<double>& model) {
             return mio::osecir::simulate_flows<double>(t0, tmax, dt, model);
         },
-        "Simulates a Secir model with flows from t0 to tmax.", py::arg("t0"), py::arg("tmax"), py::arg("dt"),
+        "Simulates an ODE SECIHURD model with flows from t0 to tmax.", py::arg("t0"), py::arg("tmax"), py::arg("dt"),
         py::arg("model"));
 
     pymio::bind_ModelNode<mio::osecir::Model<double>>(m, "ModelNode");
     pymio::bind_SimulationNode<mio::osecir::Simulation<>>(m, "SimulationNode");
     pymio::bind_ModelGraph<mio::osecir::Model<double>>(m, "ModelGraph");
-    pymio::bind_MigrationGraph<Simulation>(m, "MigrationGraph");
+    pymio::bind_MigrationGraph<mio::osecir::Simulation<>>(m, "MigrationGraph");
     pymio::bind_GraphSimulation<MigrationGraph>(m, "MigrationSimulation");
 
     //normally, std::vector is bound to any python iterable, but this doesn't work for move-only elements
```

Comment:
change line 260 to:
// These functions are in general not <del>secir</del> model dependent, only with the current config.